examples/functions/algolia-document-sync/README.md

```diff
@@ -77,13 +77,12 @@ This function is built to be compatible with any of [the official "clean" templa
    Install dependencies in the project root:
 
    ```bash
-   npm install
+   npm install dotenv
```

Comment:
When starting a clean project there is no `dotenv` in the `package.json` so I'm just being explicit. Otherwise the `sanity.blueprints.ts` file may fail to parse.